### DISCUSSION TITLE
How to let `prisma`  ignore tables which created by `laravel` ?

### DISCUSSION THREAD

**Discussion by toknT** [prisma/prisma#14226] [Q&A] [answered]
The main parts of our app using `nestjs`+`prisam` it work well, but when need to add some extra parts ( using `laravel` and laravel's third-party packages),it meet a lot of problems, because *`prisma` unable to ignore tables which created by `laravel`*. 

- ** `Laravel`  and `laravel's third-party packages` has it's own migration , it will create it's own migration data and tables** , prisma not ignore them and will create a new mgiration file for they .
-  I tried to add a extra database for laravel but the `relationship` and `foreign key`  used both by `laravel` and `nestjs`, so it's unable to do that.

**Comment by janpio** [comment, marked as answer]
Right now you can not exclude tables. A feature request for this is here: https://github.com/prisma/prisma/issues/807

But you should be able to run `db pull` initially, then use https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/baselining once to create an initial migration file for them. Then you should be able to use `migrate dev` to create new migrations that do not touch the existing tables. (When you modify these tables via Laravel you will unfortunately have to repeat this.)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
